src/module.c

```diff
@@ -6213,11 +6353,24 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
     error_as_call_replies = flags & REDISMODULE_ARGV_CALL_REPLIES_AS_ERRORS;
     va_end(ap);
 
-    c = moduleAllocTempClient();
+    if (ctx->persistent_client) {
+        c = ctx->persistent_client->client;
+    } else {
+        c = moduleAllocTempClient();
+    }
+
+    if (c->flags & CLIENT_BLOCKED) {
+        errno = -EADDRINUSE;
```

Comment:
Why is this -EADDRINUSE?

Reply:
seems appropriate, but needs to be added to RM_Call's documentation.

Reply:
errno of EADDRINUSE is ok, I'm asking why it's negative, is that a convention I just don't know about? (Or maybe just an odd typo). Yeah, also it should be documented.